### PR TITLE
Create aniruddh.is-an.app.json

### PR DESCRIPTION
**Pull request by aniruddhnishad**
### Requirements

- [ ] You have completed your website, there's no type of placeholder on the website
- [ ] The website is reachable
- [ ] The CNAME record doesn't contain `https://` or `/` 
- [ ] There is sufficient information in the `owner` field

### Link to Website

Link:

### CODE REVIEW

File: domains/aniruddh.is-an.app.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+	"$schema": "../schemas/domain.schema.json",
+	"description": "My personal project",
```
Comment (tarampampam) [inline domains/aniruddh.is-an.app.json:3]: Could you please provide a description (what is it) here? Is your pet project or something else?
```suggestion
	"description": "write-it-hete",
```

Comment (aniruddhnishad) [reply domains/aniruddh.is-an.app.json]: done

**Review by aniruddhnishad — COMMENTED** [review]
done

**Comment by tarampampam** [comment]
Sorry for the tests under your PR 😄 

Have fun:

```
❯ dig @1.1.1.1 +short aniruddh.is-an.app
172.67.181.22
104.21.64.101
```

**Pull request merged**